django_short_urls/views.py

```diff
@@ -101,21 +98,17 @@ def main(request, path):
         defaults={REF_PARAM_NAME: REF_PARAM_DEFAULT_VALUE}
     )
 
-    # FIXME: OPS-3885 - we're logging agressively to Sentry here because we don't understand some weird 404 errors on
-    # FIXME: workfor.us. We suspect they are being triggered by some pages being served directly as a proxy, perhaps.
-    if link.act_as_proxy:
-        logging.error('Page served through proxy: %s', str(link))
```

Comment:
There was no conclusion, but none of these errors appear anymore... And we have not had any end user reports of anything that could relate to this.